LiskHQ/lisk-sdk
# Update chain for on chain architecture - Closes #5609

### What was the problem?

This PR resolves #5609 

### How was it solved?

- Transaction is moved into chain.
- Basic genesis block is validate / apply is added
- Get/Set validator is added
- Default account schema is updated
- Add seed reveal validation

### How was it tested?

- Update all existing tests
- Migrate tests for genesis block validation, transaction, seed reveal
- Add test for set/get validators
- Check chain library tests description and changed implementation
